docs/man/gfx.toml.5.in

```diff
@@ -0,0 +1,182 @@
+.\"
+.TH "gfx.toml" "8" "@PACKAGE_VERSION@" "xrdp team" ""
+.SH "NAME"
+\fBgfx.toml\fR \- Configuration file for xrdp(8) graphics pipeline extension
+
+.SH "DESCRIPTION"
+
+This file configures the detailed settings of the Graphics Pipeline Extension
+for xrdp(8). The file format is TOML. See the link in the SEE ALSO section for
+TOML syntax.
+
+The file contains following sections:
+
+.TP
+\fB[codec]\fR \- configure preferred codec and encoders for \fBxrdp\fR(8).
+
+.TP
+\fB[x264]\fR \- parameters for x264 encoder.
+
+.TP
+\fB[OpenH264]\fR \- parameters for Cisco OpenH264 encoder.
+
+.LP
+All options and values are case-sensitive, with some exception, and are
+described in detail below.
+
+.SH "CODEC"
+\fB[codec]\fR section defines preferred codec order and encoder. The options
+to be specified in this section are the following:
+
+.TP
+\fBorder\fR = \fI<array>\fR
+Define the order in which codecs should be used, "H.264" and "RFX".
+Specify as an array of strings like \fB[ "H.264", "RFX ]\fR.
+Strings in the array are case-insensitive, while others are case-sensitive.
+
+.TP
+\fBh264_encoder\fR = \fI<string>\fR
+Specify a preferred H.264 encoder, \fB"x264"\fR or \fB"OpenH264"\fR.
+This parameter takes effect only when more than one encoder is
+enabled at compile time. If only one encoder is enabled, the encoder
```

Comment:
Replace "If only one encoder is enabled" with "If only one H.264 encoder is enabled."